src/nspawn/nspawn-cgroup.c

```diff
@@ -135,21 +136,22 @@ int create_subcgroup(
 }
 
 int mount_cgroups(const char *dest, bool accept_existing) {
-        const char *p;
+        _cleanup_free_ char *p = NULL;
+        _cleanup_close_ int fd = -EBADF;
         int r;
 
-        p = prefix_roota(dest, "/sys/fs/cgroup");
-
-        (void) mkdir_p(p, 0755);
+        r = chase("/sys/fs/cgroup", dest, CHASE_PREFIX_ROOT | CHASE_MKDIR_0755, &p, &fd);
+        if (r < 0)
+                return log_error_errno(r, "Failed to chase %s/sys/fs/cgroup: %m", strempty(dest));
 
-        r = path_is_mount_point_full(p, dest, AT_SYMLINK_FOLLOW);
+        r = is_mount_point_at(fd, NULL, 0);
```

Comment:
Please add a short comment for constant arguments:
`/* filename = */ NULL, /* flags = */ 0`